README.md

```diff
@@ -233,6 +234,71 @@ The above definition of the example word `foobot` would only be included if the
 - Assumes the whole software system is in a single git repository
 - Default templates assume the whole software system has a single safety classification
 
+## Audit Checklists
+
+We include several checklists for various standards. 
+These are used by the `rdm gap` command to check output documents for appropriate references to a given standard. 
+
+For example ISO62304 requires that the device be classified according to the hazard levels 
+it could present.
+This requirement occurs in section 4.3.a of the standard.
+All of the 62304 checklists will include the following:
+```
+62304:4.3.a Software safety classification: system class
+```
+The gap analysis will report whether the key word `62304:4.3.a` appears in the documents. 
```

Comment:
I think that we want "keyword" here and not "key word".

See https://ell.stackexchange.com/questions/115815/is-it-okay-to-use-key-words-as-two-separate-words-to-indicate-keywords

I would double check though, since I am not certain.

Reply:
I think "keyword" is the correct choice.